english/play-music/playmusic.md

```diff
@@ -1 +1,3 @@
 You decide to just [Play some music](https://youtu.be/IcrbM1l_BoI)
+
+But some time after, you get tired of hearing the same song, that's why you decide to listen to ["The strokes"](theStrokes/theStrokes.md) or something more popular as the music for [Colombian people](colombianMusic/colombianMusic.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.